README.md

```diff
@@ -116,6 +116,14 @@ bedrock_model = BedrockModel(
 agent = Agent(model=bedrock_model)
 agent("Tell me about Agentic AI")
 
+# Bedrock with non-streaming mode
+bedrock_model = BedrockModel(
+  model_id="us.amazon.nova-pro-v1:0",
+  streaming=False
```

Comment:
What if we just make this explicit in the example above so that folks know they can turn it on/off.

I feel like the number of examples is getting excessive